src/models/wasteActivities.model.js

```diff
@@ -93,6 +93,14 @@ module.exports = class WasteActivities {
     return selectedWasteActivity
   }
 
+  // Provides activity name in required format
+  // Reference name of an activity is included if there is one
+  // Note that the – is an en-dash
+  formattedActivityName (activity) {
+    const { item, referenceName } = activity
+    return referenceName ? `${item.itemName} – ${referenceName}` : item.itemName
+  }
+
```

Comment:
Not sure why this is here - potentially a premature refactoring?

Not a big deal, but thought I'd mention it.

Reply:
Yes probably a bit premature -- I broke it out into its own function with the intention of using it in the tasklist to ensure a consistent format for activity names, but I ended up using the output of `wasteActivityNames` for that anyway so I guess there's no real need for this to be a separate function at the moment.